### DISCUSSION TITLE
2005/12/05/%E7%94%9F%E6%B4%BB%E9%9A%8F%E7%AC%94

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#63] [Announcements]
# 2005/12/05/%E7%94%9F%E6%B4%BB%E9%9A%8F%E7%AC%94

新的生活随笔文章列表：http://ruanqizhen.wordpress.com/essay/

https://blog.qizhen.xyz/2005/12/05/%E7%94%9F%E6%B4%BB%E9%9A%8F%E7%AC%94

**Comment by ruanqizhen** [comment]
之前博客下的留言：

Moon
2008年06月24日 14:19 编辑
一直以为阮老大是个技术超产,不曾想还是个多产的作者
失敬失敬
今日得以拜读大作
胜做十年LV
呵呵

回复
Unknown
2005年12月27日 21:52 编辑
我发现读paper真的是一个很好的排遣烦闷平静心情的方式…

回复
